model/model_training/custom_datasets/oasst_dataset.py

```diff
@@ -35,6 +38,10 @@ def load_oasst_export(
     generator = Generator()
     generator.manual_seed(manual_seed)
 
+    if use_hf_dataset and hf_dataset_name:
+        hf_dataset = load_dataset(hf_dataset_name)
+        return hf_dataset["train"], hf_dataset["validation"]
```

Comment:
The format between trees and messages in the HF dataset differs greatly .. some conversion is required to restore the OA format.

Reply:
any tips/suggestions about it? first time working on an issue